src/client/components/ModalViewCancelEvent/ModalViewCancelEvent.stories.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+import { ModalCancelEvent } from './ModalViewCancelEvent';
+
+export default {
+  title: 'Modal Cancel Event',
+};
+
+export const CancelEventModal = () => (
+  <>
```

Comment:
I know it works well but just curious why you decided to use  `<>` rather than a `<div>` .

Reply:
`<>` doesn't introduce a new DOM element that might affect your styling.